bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/presentation-download-dropdown/component.jsx

```diff
@@ -121,39 +107,47 @@ class PresentationDownloadDropdown extends PureComponent {
             closeModal();
             handleDownloadingOfPresentation('Annotated');
           },
-        },
-      );
+        });
+      } else {
+        this.menuItems.push({
+          key: this.actionsKey[1],
+          dataTest: 'sendAnnotatedDocument',
+          label: intl.formatMessage(intlMessages.sendAnnotatedDocument),
```

Comment:
Since it may not contain annotations anymore, I think a better label for it would be "Send out a download link of the current presentation state."